src/components/CurriculumComponents/CurriculumDownloads/CurriculumDownloads.test.tsx

```diff
@@ -62,9 +62,12 @@ describe("Component - Curriculum Header", () => {
     await userEvent.click(getByTestId("checkbox-download"));
     await userEvent.click(getByTestId("termsCheckbox").querySelector("label")!);
     await userEvent.click(getByTestId("loadingButton"));
-    await waitFor(() => {
-      expect(getByTestId("downloadSuccess")).toBeInTheDocument();
-    });
+    await waitFor(
+      () => {
+        expect(getByTestId("downloadSuccess")).toBeInTheDocument();
+      },
+      { interval: 1000, timeout: 10000 },
+    );
```

Comment:
does this change at all if you import `waitFor` from testinglibrary/react instead of dom?

Reply:
No still seems to be failing without the options, I think it just needs more time to render the component which is why it wasn't working